tests/test_core.py

```diff
@@ -313,6 +319,11 @@ def test_hello_world(self):
     # must not emit this unneeded internal thing
     self.assertNotContained('EMSCRIPTEN_GENERATED_FUNCTIONS', read_file('test_hello_world.js'))
 
+  @also_with_wasmfs
```

Comment:
Sorry if I wasn't clear, I was suggesting adding a new decorator to the existing test. That would avoid adding a new test entirely, just one line for the decorator. (Well, and also defining the decorator the first time.)

Reply:
Oh apologies. I will delete the new test I added!